e2e/go.mod

```diff
@@ -0,0 +1,11 @@
+module e2e
```

Comment:
@gianarb wdyt about what we proposed here about having a separated module to avoid pushing the postgres dependency to downstream consumers?

Reply:
Ok. Readme added.